src/angular-app/languageforge/lexicon/editor/editor.component.ts

```diff
@@ -680,10 +680,11 @@ export class LexiconEditorController implements angular.IController {
 
   getFontFamilyForPrimaryListItemForDisplay(entry: LexEntry) {
     if (!this.getSortableValue(this.lecConfig, entry)) return '';
+
     // FIXME this is not always accurate, given the complexity in get EditorDataService#getSortableValue
-    return this.lecConfig.inputSystems[
-      (this.lecConfig.entry.fields.lexeme as LexConfigMultiText).inputSystems[0]
-    ].cssFontFamily;
+    const lexemeField = this.lecConfig.entry.fields.lexeme as LexConfigMultiText;
+    const inputSystem = lexemeField.inputSystems.find(system => !system.includes('-audio'));
```

Comment:
helpful refactor into multiple variables that are easier to read.

Note: this also filters out audio writing systems.